lms/envs/common.py

```diff
@@ -1247,7 +1247,8 @@
     'js/vendor/requirejs/require.js',
     'js/RequireJS-namespace-undefine.js',
     'js/vendor/URI.min.js',
-    'js/vendor/backbone-min.js'
+    'js/vendor/backbone-min.js',
+    'js/vendor/afontgarde/modernizr.fontface-generatedcontent.js',
```

Comment:
Yes. I need these files now and since the UXPL isn't fully integrated, that will need to be a separate pull request. I will try removing this instance of Modernizr though and see if the icons (used in the video player) still work as expected.